sql-gremlin/src/main/java/org/twilmes/sql/gremlin/adapter/converter/SqlTraversalEngine.java

```diff
@@ -151,7 +163,8 @@ private static void appendGraphTraversal(final String table, final String column
             }
         } else {
             // It's this vertex/edge.
-            if (columnName.toLowerCase(Locale.getDefault()).startsWith(gremlinTableBase.getLabel().toLowerCase(Locale.getDefault()))) {
+            if (columnName.toLowerCase(Locale.getDefault())
+                    .startsWith(gremlinTableBase.getLabel().toLowerCase(Locale.getDefault()))) {
```

Comment:
Locale.getDefault() here is redundant. No need for it.

Reply:
This was forced from a checkstyle.